Examples/draggable-graph.lisp

```diff
@@ -24,49 +24,39 @@
 
 (define-application-frame draggable-graph-demo () ()
   (:pane (make-pane 'application-pane
-		    :width :compute
-		    :height :compute
-		    :display-function 'generate-graph
-		    :display-time t)))
+                    :width :compute
+                    :height :compute
+                    :display-function 'generate-graph
+                    :display-time t)))
 
 (defun generate-graph (frame pane)
   (declare (ignore frame))
   (format-graph-from-roots
    (list (find-class 'number))
    (lambda (object stream)
-     (present (class-name object)
-	      (presentation-type-of object)
-	      :stream stream))
+     (present (class-name object) (presentation-type-of object)
+              :stream stream))
```

Comment:
I don't understand why did you change one parameter per line style to "two parameters in the first and one remaining in a separate line", I'm fine with having all in one line or putting everything in separate lines but such mixed style doesn't convince me (especially here where we don't have that many arguments,  I could understand two obligatory arguments in first line and keyword arguments in next (one by one) if there were more of them.